CommentUpdater/src/main/java/org/eupathdb/sitesearch/data/comments/ApolloCommentUpdaterCli.java

```diff
@@ -0,0 +1,29 @@
+package org.eupathdb.sitesearch.data.comments;
+
+public class ApolloCommentUpdaterCli extends CommentUpdaterCli {
+
+  public static void main(String[] args) throws Exception {
+    UserCommentUpdaterCli cli = new UserCommentUpdaterCli();
+    cli.init(new ApolloCommentSolrDocumentFields(), new ApolloCommentUpdaterSql());
```

Comment:
Why not pass these to the abstract class constructor rather than init()?  Looks like the class is in an invalid state before init() is called.